srcpkgs/cross-x86_64-w64-mingw32/template

```diff
@@ -1,14 +1,14 @@
-_binutils_version=2.29.1
-_gcc_version=8.3.0
+_binutils_version=2.32
+_gcc_version=8.2.0
 _gmp_version=6.1.2
 _mpfr_version=4.0.1
 _mpc_version=1.1.0
 _isl_version=0.19
-_mingw_version=5.0.4
+_mingw_version=6.0.0
 
 pkgname=cross-x86_64-w64-mingw32
 version=${_mingw_version}
-revision=5
+revision=3
```

Comment:
Set revision to 1. Also, squash your commits into one and make sure that the is no merge commit